paths_cli/parameters.py

```diff
@@ -109,6 +109,15 @@ def get(self, storage, names):
     store='schemes'
 )
 
+MULTI_INIT_SNAP = OPSStorageLoadMultiple(
+    param=Option('-f', '--init-frame',
+                 help="identifier for potential initial snapshots"),
+    store='snapshots',
+    value_strategies=[GetByName('tags'), GetByNumber('snapshots')],
+    none_strategies=[GetOnlySnapshot(),
+                     GetPredefinedName('tags', 'initial_snapshot')]
+)
+
```

Comment:
Is this actually used somewhere?

Reply:
No; the original idea in this branch was to write something that would be a single command to set up an entire network (which could require multiple initial snapshots). But that's a little too messy, and so I'd rather have the user set up one transition at a time.

Removed `MULTI_INIT_SNAP` in  8352adc.